packages/app/app/store/enhancers/syncStorage.js

```diff
@@ -8,11 +8,16 @@ export default function(paths) {
       initialState = undefined;
     }
 
-    let persistedState, finalInitialState;
+    // retrieve initial states from reducers, to apply prior to merging electron-store persisted-data
```

Comment:
Why is this needed?

We already have a mechanism for applying default values to settings that weren't touched, unless this is needed for a different reason.

Reply:
From what I remember, without the merging of `reducerInitialStates` beforehand, the supplying of data for `state.local` within the `persistedState` object, was causing the "default value" for the `state` parameter in the `local` reducer to be ignored. (since the reducer function's default-value is only used when `undefined` is passed for that `state` parameter)

I'll double-check this though.

Reply:
Yep, the description above is correct; without that `reducerInitialState` merging, the default values for the fields in `state.local` (in `reducers/local.js`) are excluded on program start, due to there being data for that root `state.local` field from the `persistedState` source.